index.html

```diff
@@ -13,33 +13,33 @@
     <div class = "header-logo">
      <img src = img/logo.png alt = "Кваст">
     </div>
-     <ul class = "main-nav ">
-      <li><a href = "#">История</a></li>
-      <li><a href = "#">Производство</a></li>
-      <li><a href = "#">Ассортимент</a></li>
-      <li><a href = "#">Где купить</a></li>
+     <ul class = "main-nav">
+      <li><a href = "#history">История</a></li>
+      <li><a href = "#features">Производство</a></li>
+      <li><a href = "#prise-list">Ассортимент</a></li>
+      <li><a href = "#form">Где купить</a></li>
      </ul>
-   </div> 
-  <div class = "promo change">
-    Крафтовый квас<br>
-      всему голова!
+   </div>
+          <div class="promo" id="box1" style="display: block;">Крафтовый квас<br>всему голова!</div>
+          <div class="promo" id="box2" style="display: none;">Крафтовый квас<br> всему голова, а <br>некоторым и тело!</div>
+          <div class="promo" id="box3" style="display: none;">Крафтовый квас<br> всему голова, а <br>хлеб тоже!</div>
```

Comment:
Можно добавить классу promo в css свойство display: none, и добавить еще один класс active, у которого будет стиль 
.promo.active{ display: block }
Тогда будет выглядеть вот так:
```suggestion
          <div class="promo active" id="box1">Крафтовый квас<br>всему голова!</div>
    Крафтовый квас<br>
          <div class="promo" id="box2">Крафтовый квас<br> всему голова, а <br>некоторым и тело!</div>
      всему голова!
          <div class="promo" id="box3">Крафтовый квас<br> всему голова, а <br>хлеб тоже!</div>
```